client/app/views/templates/view_auth.jade

```diff
@@ -6,6 +6,10 @@ block region
     label.with-input(for="#{prefix}-password", aria-describedby="#{prefix}-password-tip")
         span= t(prefix + ' password')
         input(id="#{prefix}-password" name="password" type="password" autofocus=true)
+    if otp
+        label.with-input(for="#{prefix}-otp")
+            span= t(prefix + ' otp')
+            input(id="#{prefix}-otp" name="otp" type="text" autofocus=true autocomplete="off")
```

Comment:
Can you use `type="number'` instead of `type="text"` ? This way users on mobile devices will get a numbers keypad rather than a text one.